hw/ip/spi_host/doc/_index.md

```diff
@@ -502,6 +547,12 @@ This means that when copying bytes into {{< regref "DATA" >}} from unaligned fir
 Full-word instructions should however be used whenever possible, because each write consumes a full word of data in the TX FIFO regardless of the instruction size.
 Smaller writes will thus make inefficient use of the TX FIFO.
 
+Filtering out disabled bytes consumes clock cycles in the data pipeline, and can create bubbles in the transmission of SPI_DATA.
+In the worst case, such bubbles can also be interpreted as transient underflow conditions in the TX FIFO, and could trigger spurious interrupts.
+The longest delays occur whenever a word is loaded into the TX FIFO with only one byte enabled.
+
+The SPI_HOST IP silently ignores zero-byte writes to the {{< regref "DATA" >}} window, should they ever occur.
```

Comment:
Are these caused by a TL transaction with a zero width? Is that even possible? (Sounds like something we should probably have assertions about at the system and/or block level)

Reply:
I don't know why they would be caused: a rabid wild Ibex?  More likely a DV block would just put this in the queue trying to break it, because UVC's are literal chaos machines.   It's not an expected condition, just another element in the universe of weird cases that we have to do something about.  Assertions are a valid option.  I can change this.

Reply:
Sounds reasonable to me. I guess there's a security question here, but I don't suppose that someone messing up the behaviour of our SPI host is really a big deal.

Reply:
I can open an issue to get Security feedback on this.   The big hammer approach would be to make a class of error interrupts for this unusual thing.  (There's plenty of space in the error register), I'll propose that in the issue.

However you are correct the biggest issue here is that this causes a brief stall and slows the transaction down by a clock cycle.  However if this could happen and verification never checks the DUT response, maybe that *is* some to be concerned about?

Reply:
Yep, I agree. It makes sense to specify. Maybe we should add a note in this sort of situation to say something like "We don't expect to use this but..."